core/Plugin/Controller.php

```diff
@@ -476,25 +476,18 @@ protected function getLastUnitGraphAcrossPlugins($currentModuleName, $currentCon
      */
     protected function getGraphParamsModified($paramsToSet = array())
     {
-        if (!isset($paramsToSet['period'])) {
-            $period = Common::getRequestVar('period');
-        } else {
-            $period = $paramsToSet['period'];
+        try {
+            $period = isset($paramsToSet['period']) ?? Piwik::getPeriod();
```

Comment:
That can't work, as period will be `true` if `$paramsToSet['period']` has a value.
```suggestion
            $period = $paramsToSet['period'] ?? Piwik::getPeriod();
```